src/ConstructiveSolidGeometry/ConstructiveSolidGeometry.jl

```diff
@@ -72,9 +72,11 @@ module ConstructiveSolidGeometry
     rotation(p::AbstractPrimitive) = p.rotation
     origin(p::AbstractPrimitive) = p.origin
     _transform_into_global_coordinate_system(pt::CartesianPoint, p::AbstractPrimitive) = (rotation(p) * pt) + origin(p)
+    _transform_into_global_coordinate_system(pt::CartesianVector, p::AbstractPrimitive) = rotation(p) * pt
     _transform_into_global_coordinate_system(pts::AbstractVector{<:CartesianPoint}, p::AbstractPrimitive) =
         broadcast(pt -> _transform_into_global_coordinate_system(pt, p), pts)
     _transform_into_object_coordinate_system(pt::CartesianPoint, p::AbstractPrimitive) = inv(rotation(p)) * (pt - origin(p)) 
+    _transform_into_object_coordinate_system(pt::CartesianVector, p::AbstractPrimitive) = inv(rotation(p)) * pt
```

Comment:
These are the new methods for transforming `CartesianVectors` (just without the translation)